Python/pywarpx/_libwarpx.py

```diff
@@ -74,8 +74,13 @@ def _get_package_root():
 
 try:
     libwarpx = ctypes.CDLL(os.path.join(_get_package_root(), libname))
-except OSError:
-    raise Exception('"%s" was not installed. It can be installed by running "make" in the Python directory of WarpX' % libname)
+except OSError as e:
+    value = e.args[0]
+    if any(so in value for so in ["libwarpx.2d.so", "libwarpx.3d.so"]):
+        raise Exception('"%s" was not installed. It can be installed by running "make" in the Python directory of WarpX' % libname)
+    else:
+        print("Failed to load the libwarpx shared object library")
+        print(e)
```

Comment:
Would it be better to terminate the program here? If it's not terminated here (via an exception or `sys.exit()`), then it will crash when it gets to line 86.

Reply:
```suggestion
        raise
```

Reply:
@MKieburtz Yep, using `raise` escalates the error and keeps all the traceback info (`print` actually discards a lot of info), so raise is what you want.

Reply:
Just add `raise` after the print statements.